DESCRIPTION

```diff
@@ -20,7 +20,7 @@ Description: Use the paged media properties in CSS and the JavaScript
   pages. Each page can have its page size, page numbers, margin boxes, and
   running headers, etc. Applications of this package include books, letters,
   reports, papers, business cards, resumes, and posters.
-Imports: rmarkdown (>= 1.16), bookdown (>= 0.8), htmltools, jsonlite, later (>= 1.0.0),
+Imports: rmarkdown (>= 2.7), bookdown (>= 0.8), htmltools, jsonlite, later (>= 1.0.0),
```

Comment:
```suggestion
Imports: rmarkdown (>= 2.8), bookdown (>= 0.8), htmltools, jsonlite, later (>= 1.0.0),
```

I thought we needed 2.8 (https://pkgs.rstudio.com/rmarkdown/news/index.html#rmarkdown-28). This is the one supporting the SASS files in `css`